### ISSUE TITLE
Use the newly provided CDN for DigitalOcean's Spaces

### ISSUE THREAD

**Issue by vlki** [Demagog2/demagog#245]
https://blog.digitalocean.com/spaces-now-includes-cdn/

**Comment by vlki** [comment]
@vaclavbohac Yes, I enabled it and hoped that it would work out of the box, but that's not the case. To serve files via CDN, we have to serve them from hostname `demagog-cdn.ams3.cdn.digitaloceanspaces.com` instead of `demagog-cdn.ams3.digitaloceanspaces.com`. You can see that also in the Spaces UI:
<img width="569" alt="screen shot 2018-10-02 at 11 08 56" src="https://user-images.githubusercontent.com/116486/46339772-f9b7bc80-c633-11e8-9f8a-a0ee18bd5f52.png">

So it is about updating the backend code to use the CDN hostname.

**Comment by vaclavbohac** [comment]
I see, I will take care of it then 🙂
